src/webgpu/gpu_test.ts

```diff
@@ -180,7 +177,7 @@ export class GPUTest extends Fixture<GPUTestSubcaseBatchState> {
 
   // Should never be undefined in a test. If it is, init() must not have run/finished.
   private provider: DeviceProvider = undefined!;
```

Comment:
should this one also be typed `DeviceProvider | undefined` ? since above we also do `this.provider?.then`

and then, `get device(): Device` can also have the assert